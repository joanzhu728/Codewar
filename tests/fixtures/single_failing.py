import codewars_test as test


@test.describe("group 1")
def group_1():
    @test.it("test 1")
    def test_1():
        test.assert_equals(1, 2)
