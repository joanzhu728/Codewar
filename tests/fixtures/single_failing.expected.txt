
<DESCRIBE::>group 1

<IT::>test 1

<FAILED::>1 should equal 2

<COMPLETEDIN::>0.00

<COMPLETEDIN::>0.01
